FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/tcp_mem_stats.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * tcp_mem_stats.h
+ */
+ 
+ 
+#ifndef TCP_MEM_STATS_H
+
+#define TCP_MEM_STATS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum xTCP_MEMORY
+{
+	tcpSOCKET_TCP,
+	tcpSOCKET_UDP,
+	tcpSOCKET_SET,
+	tcpSEMAPHORE,
+	tcpRX_STREAM_BUFFER,
+	tcpTX_STREAM_BUFFER,
+	tcpNETWORK_BUFFER,
+} TCP_MEMORY_t;
+
+#if( ipconfigUSE_TCP_MEM_STATS != 0 )
+
+	void vTCPMemStatCreate( TCP_MEMORY_t xMemType, void *pxObject, size_t uxSize );
+
+	void vTCPMemStatRemove( void *pxObject );
```

Comment:
Did you mean to declare `vTCPMemStatsDelete` here?
Because `vTCPMemStatsRemove` is neither defined nor used anywhere. Line 37 seems correct since it uses a function defined in tcp_mem_stats.c.

Reply:
Oops yes, you are correct. In an earlier version, I called them "create/remove", which are not really opposite.
Now it is "create/delete". I changed the header file, thank.